jobs/delete-version.groovy

```diff
@@ -30,6 +30,10 @@ spaces. Must exactly match the name of an existing version (e.g.
     """The modules to delete, comma-separated.  Must not be empty.""",
     ""
 
+// Buildmaster triggers this job once for each service every 15 minutes.
+// If we disallow concurrent builds, the job queue can get backed up.
+).allowConcurrentBuilds(
```

Comment:
Other files have this at the top of the chain. Might be best to keep that consistent.